import/index.template.html

```diff
@@ -495,9 +573,29 @@
               document.body.style.backgroundColor = validStyles.backgroundColor;
             }
 
+            if (validStyles.labelColor) {
+              document.getElementById("mnemonic-label").style.color =
+                validStyles.labelColor;
+            }
```

Comment:
I don't really understand where `labelColor` can come from. It's not in the list of supported styles:
https://github.com/tkhq/sdk/blob/c214672c4f6115b27d41bd460c2162e033b0e84f/packages/iframe-stamper/src/index.ts#L107-L129